packages/core/src/components/Search/Search.tsx

```diff
@@ -116,8 +118,10 @@ const Search = forwardRef(
         inputRole={searchResultsContainerId ? "combobox" : undefined}
         aria-label={inputAriaLabel}
         aria-busy={loading}
-        aria-owns={searchResultsContainerId}
+        aria-controls={ariaExpanded ? searchResultsContainerId : undefined}
         aria-activedescendant={currentAriaResultId}
+        aria-haspopup={ariaHasPopup}
```

Comment:
from what I know, it basically says that the Search input can trigger an open of a popup
I'm not sure we have such case
in Combobox, the Search is part of the dialog, and does not trigger an open of a popup